requirements/edx/base.txt

```diff
@@ -50,7 +50,7 @@ git+https://github.com/Stanford-Online/xblock-free-text-response@release/v0.4.0#
 -e git+https://github.com/Stanford-Online/xblock-mufi.git@release/0.2.1#egg=xblock_mufi-master
 -e git+https://github.com/edx-solutions/xblock-ooyala.git@32d52edaa820dbdbf846d8a84f6bccbf0b9c8218#egg=xblock_ooyala_player-master
 git+https://github.com/Stanford-Online/xblock-platform-tour.git@v1.0.3#egg=xblock-platform-tour==1.0.3
-git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
+git+https://github.com/open-craft/xblock-poll@3a193ab1ee5443342aff356797b743e319f45995#egg=xblock-poll==1.8.7
```

Comment:
shouldn't this be https://github.com/open-craft/xblock-poll/commit/6abacb5f506ee6636723935f4116af29bc53abda

Reply:
Good question, I "got greedy" and went with `master`...

So I poked at the commit list going back from `master` and didn't notice anything major, mostly translations and such. I didn't see anything that seemed backward incompatible, no migrations, etc.

So we can go either way; pin it directly to that fix or try just taking master.
Preference?

Reply:
Ok, looks fine to take latest master 👍